static/app/components/stream/group.tsx

```diff
@@ -307,116 +306,89 @@ function BaseGroupRow({
     withChart && group && group.filtered && statsPeriod && useFilteredStats
   );
 
+  const groupCategoryCountTitles: Record<IssueCategory, string> = {
+    [IssueCategory.ERROR]: t('Error Events'),
+    [IssueCategory.PERFORMANCE]: t('Transaction Events'),
+  };
+
   const groupCount = !defined(primaryCount) ? (
     <Placeholder height="18px" />
   ) : (
-    <DeprecatedDropdownMenu isNestedDropdown>
-      {({isOpen, getRootProps, getActorProps, getMenuProps}) => {
-        const topLevelCx = classNames('dropdown', {'anchor-middle': true, open: isOpen});
-
-        return (
-          <GuideAnchor target="dynamic_counts" disabled={!hasGuideAnchor}>
-            <span {...getRootProps({className: topLevelCx})}>
-              <span {...getActorProps({})}>
-                <div className="dropdown-actor-title">
-                  <PrimaryCount value={primaryCount} />
-                  {secondaryCount !== undefined && useFilteredStats && (
-                    <SecondaryCount value={secondaryCount} />
-                  )}
-                </div>
-              </span>
-              {useFilteredStats && (
-                <StyledDropdownList
-                  {...getMenuProps({className: 'dropdown-menu inverted'})}
-                >
-                  {group.filtered && (
-                    <Fragment>
-                      <StyledMenuItem to={getDiscoverUrl(true)}>
-                        <MenuItemText>
-                          {queryFilterDescription ?? t('Matching search filters')}
-                        </MenuItemText>
-                        <MenuItemCount value={group.filtered.count} />
-                      </StyledMenuItem>
-                      <MenuItem divider />
-                    </Fragment>
-                  )}
-
-                  <StyledMenuItem to={getDiscoverUrl()}>
-                    <MenuItemText>{t('Total in %s', summary)}</MenuItemText>
-                    <MenuItemCount value={group.count} />
-                  </StyledMenuItem>
-
-                  {group.lifetime && (
-                    <Fragment>
-                      <MenuItem divider />
-                      <StyledMenuItem>
-                        <MenuItemText>{t('Since issue began')}</MenuItemText>
-                        <MenuItemCount value={group.lifetime.count} />
-                      </StyledMenuItem>
-                    </Fragment>
-                  )}
-                </StyledDropdownList>
-              )}
-            </span>
-          </GuideAnchor>
-        );
-      }}
-    </DeprecatedDropdownMenu>
+    <GuideAnchor target="dynamic_counts" disabled={!hasGuideAnchor}>
+      <Tooltip
+        disabled={!useFilteredStats}
+        isHoverable
+        title={
+          <CountTooltipContent>
+            <h4>{groupCategoryCountTitles[group.issueCategory]}</h4>
```

Comment:
Do we want a fallback here? We won't be adding issue categories frequently, but when the profiling issues come in this will be headerless (which maybe is okay)

Reply:
I made sure the typescript type would fail if a title was not added, so I think it's OK